src/docfx/build/context/Cache.cs

```diff
@@ -4,16 +4,11 @@
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
-using System.IO;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal class Cache
     {
-        private readonly ConcurrentDictionary<string, Lazy<(List<Error>, JToken)>> _tokenCache
```

Comment:
what are we benefiting by moving the JSON/YAML cache from `Cache.cs` to `Input.cs`?
Or why do we leave TOC cache in `Cache.cs`?

Reply:
Just noticed #5211, so the idea is to get rid of `Cache.cs`